app/controllers/user_projects_controller.rb

```diff
@@ -13,14 +13,15 @@ def index
     @users = @project.user_projects
 
     respond_to do |format|
-      #format.html
-      format.json {
-        render :json => {
-          :html => render_to_string({
-            :partial => "index.html.erb"
-          })
+      format.json do
+        render json: {
+          html: render_to_string(
+            partial: 'index.html.erb'
+          ),
+          project_id: @project.id,
+          counter: @project.users.count
```

Comment:
Same here, add comment `# Used for counter badge` or something along those lines.